realm/realm-library/src/main/java/io/realm/ManagedMapManager.java

```diff
@@ -225,7 +225,7 @@ private void checkForAddRemoveListener(@Nullable Object listener, boolean checkL
     }
 
     @Override
-    boolean containsKeyInternal(@Nullable Object key) {
+    boolean containsKeyInternal(Object key) {
```

Comment:
Why is the `@Nullable` annotation removed, when the argument can in fact be nullable (or at least it is checked for it)?

Reply:
It is because it was misplaced, the argument is not nullable.